openedx/features/edly/utils.py

```diff
@@ -25,20 +25,24 @@ def user_has_edly_organization_access(request):
     Returns:
         bool: Returns True if User has Edly Organization Access Otherwise False.
     """
-    if not request.user.edly_profile:
-        return False
 
-    current_site = request.site
-    edly_sub_org = EdlySubOrganization.objects.filter(
-        Q(lms_site=current_site) |
-        Q(studio_site=current_site)
-    ).first()
+    try:
```

Comment:
Revert these changes back

Reply:
@zubair-arbi since there is a `OneToOne` relation of site and `EdlySubOrganization`, shouldn't we `get` it instead of filtering?

Reply:
Yes, that's fine